packages/visual-editor/src/hooks/useComponentProps.ts

```diff
@@ -99,18 +103,31 @@ export const useComponentProps = ({
     const extractedProps = Object.entries(definition.variables).reduce(
       (acc, [variableName, variableDefinition]) => {
         const variableMapping = node.data.props[variableName];
+
         if (!variableMapping) {
           return {
             ...acc,
             [variableName]: variableDefinition.defaultValue,
           };
         }
 
+        /* variableMapping {
+          type: 'DesignValue',
+          valuesByBreakpoint: {
+            desktop: '300px',
+            // tablet: '300px',
+            // mobile: '300px',
+          },
+        }
+        */
```

Comment:
Was this added accidentally for debugging purposes or meant for documenting the shape of `variableMapping`? If it's the latter, would it help to have proper types instead?

Reply:
This was for testing and deubugging purpose.